spot_wrapper/wrapper.py

```diff
@@ -1893,6 +1937,110 @@ def _navigate_to(self, *args):
         else:
             return False, "Navigation command is not complete yet."
 
+    @try_claim
+    def _navigate_to_dynamic(self, *args):
+        """Navigate to a specific waypoint. Can change velocity and pause/cancel navigation (see handle_set_navigate_to_params in spot_ros2)"""
```

Comment:
Again, better to refer to how the pause/cancel is controlled by referencing wrapper commands, or how to control it otherwise.